<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accountWidget</class>
 <widget class="QWidget" name="accountWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(44, 60, 75);
border: 0px;
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>-10</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/Ellipse.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/profileButton.png</normaloff>../../../Downloads/profileButton.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>151</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="wardenfort.qrc">:/wardenfort/frame.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: white;</string>
      </property>
      <property name="text">
       <string>PERSONAL DETAILS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; color: white;</string>
      </property>
      <property name="text">
       <string>WardenFort uses this information to verify your identity and to keep our community safe. </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; color: white;</string>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; color: white;</string>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>263</y>
        <width>220</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
border-radius: 15px; 
background-color: #191F32;
padding-left: 10px;
	color: rgb(255, 255, 255);
}

</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>263</y>
        <width>220</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
border-radius: 15px; 
background-color: #191F32;
padding-left: 10px;
	color: rgb(255, 255, 255);
}

</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; color: white;</string>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>220</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
border-radius: 15px; 
background-color: #191F32;
padding-left: 10px;
	color: rgb(255, 255, 255);
}
QComboBox::down-arrow { 
image: none; 
border: none;
border-radius: 10px;
}
QComboBox::drop-down { 
width: 0px; 
}

</string>
      </property>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>220</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QDateEdit {
border-radius: 15px; 
background-color: #191F32;
padding-left: 10px;
	color: rgb(255, 255, 255);
}
QDateEdit::down-button,QDateEdit::up-button { 
width: 0; height: 0; border: none; 
}

</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>310</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; color: white;</string>
      </property>
      <property name="text">
       <string>Date of Birth</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; color: white;</string>
      </property>
      <property name="text">
       <string>Email Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>400</y>
        <width>220</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
border-radius: 15px; 
background-color: #191F32;
padding-left: 10px;
	color: rgb(255, 255, 255);
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>460</y>
        <width>220</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
border-radius: 15px; 
background-color: #191F32;
padding-left: 10px;
	color: rgb(255, 255, 255);
}

</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; color: white;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>400</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #01FDF5;
color: black;
border-radius: 15px;</string>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>460</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #01FDF5;
color: black;
border-radius: 15px;</string>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>360</y>
        <width>111</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>550</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; color: white;</string>
      </property>
      <property name="text">
       <string>WardenFort uses this information to verify your identity and to keep our community safe. </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>290</y>
        <width>111</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>pushButton</zorder>
     <zorder>label_6</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>lineEdit</zorder>
     <zorder>lineEdit_2</zorder>
     <zorder>label_10</zorder>
     <zorder>comboBox</zorder>
     <zorder>dateEdit</zorder>
     <zorder>label_11</zorder>
     <zorder>label_12</zorder>
     <zorder>lineEdit_3</zorder>
     <zorder>lineEdit_4</zorder>
     <zorder>label_13</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>label_15</zorder>
     <zorder>label_16</zorder>
     <zorder>label_18</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="wardenfort.qrc"/>
 </resources>
 <connections/>
</ui>
